src/components/ArticleList.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from "react";
+import { getArticles } from "../api.js";
+import ArticleCard from "./ArticleCard";
+import SortFilterBar from "./SortFilterBar";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [sortBy, setSortBy] = useState("created_at");
+  const [filterBy, setFilterBy] = useState("");
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    getArticles()
+      .then((data) => {
+        let filteredArticles = data.articles;
+
+        if (filterBy) {
+          filteredArticles = filteredArticles.filter(
```

Comment:
filter returns a new array so you shouldn't need to reassign filteredArticles...